package.json

```diff
@@ -48,7 +48,7 @@
     "publishCmd": "npm publish",
     "publishDevCmd": "npm publish --tag ${devTag}",
     "versionCmd": "npm version --no-git-tag-version ${version}",
-    "cleanCmd": "git checkout -- package.json",
+    "cleanCmd": "git checkout -- package.json package-lock.json",
```

Comment:
@atisayajain I was testing this out today and found that it isn't as robust as needed. Take for example the case where no package-lock.json exists or if it is there but is not tracked by git. In both of those cases it an error will be thrown by git. Instead we should check first if the file is being tracked by git before trying to clean it up. 

One option:
- add a new command to check if a file is being tracked by version control (see: https://stackoverflow.com/questions/2405305/how-to-tell-if-a-file-is-git-tracked-by-shell-exit-code)
- add another option that contains a list of files to clean
- update the `cleanCmd` to take a token that will be replace by each of the above files
- update `publish.clean` to iterate over the list of files to clean. It will first need to check if the file is being versioned and then call the cleanCmd on it.